Header header
uint8 DroneState
uint8 DOCKED= 0
uint8 FLYING= 1
uint8 LANDING= 2